sttp/build.sbt

```
@@ -10,11 +10,11 @@ val `sttp-client` =
       publishSettings,
       `scala 2.12 to 2.13`,
       name := "sttp-client",
-      version := "3.0.0+n",
+      version := "3.0.0",
       versionPolicyIntention := Compatibility.BinaryAndSourceCompatible,
```

We need to bump the version to 4.0.0, and to declare that we will break the binary compatibility with 3.0.0

```suggestion
      version := "3.0.0+n",
      versionPolicyIntention := Compatibility.None,
```